src/mos.c

```diff
@@ -957,10 +1494,79 @@ UINT24	mos_DIR(char * path) {
 // - FatFS return code
 // 
 UINT24 mos_DEL(char * filename) {
-	FRESULT	fr;	
-	
-	fr = f_unlink(filename);
-	return fr;
+    FRESULT fr = FR_INT_ERR;

```

Comment:
I'm slightly nervous about this version of delete, essentially because it's incredibly powerful, and there are no guard-rails

previously you could delete a single file.  with this new version doing `*delete *` will delete *everything* in a directory.  there is no verification with the user that they _really_ want to do that.

it feels like as a bare minimum if a pattern is being used we should verify each individual delete with the user.

Reply:
this function has been reverted to a plain delete

the star command handler for delete includes this wildcard matching functionality, and has an interactive confirmation for deletes